### DISCUSSION TITLE
Map rotation 45°, 135°, 225°, 315°

### DISCUSSION THREAD

**Discussion by adrgumula** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#520] [Ideas]
I want to define zones in the following apparent:
![image](https://user-images.githubusercontent.com/70687019/208298433-6b18a6a8-d84b-46b7-9b79-00ab8c7c529e.png)

However, it would be much easier (natural) to have possibility to rotate map 45° (and variation of that).

**Comment by PiotrMachowski** [comment]
You can create a feature request in Map Extractor
